pkg/test/skip/skip_test.go

```diff
@@ -0,0 +1,25 @@
+package skip_test
+
+import (
+	"os"
+	"testing"
+
+	. "github.com/smartystreets/goconvey/convey"
+
+	tskip "zotregistry.io/zot/pkg/test/skip"
+)
+
+// for code coverage.
+func TestSkipS3(t *testing.T) {
+	Convey("Test if S3MOCK_ENDPOINT env is unset", t, func() {
+		os.Clearenv()
```

Comment:
I think this one removes everything from the environment, including stuff we need. I think we need to set everything back in a defer.